src/llmq/utils.cpp

```diff
@@ -660,6 +660,11 @@ bool IsInstantSendLLMQTypeShared()
     return false;
 }
 
+bool IsPlatformLLMQType(Consensus::LLMQType llmqType)
+{
+    return Params().GetConsensus().llmqTypePlatform == llmqType;
+}
+
```

Comment:
It'd be more readable in certain places if we added a wrapper / second name here something like `IsLLMQTypeHPMNOnly`